lib/models/message.ts

```diff
@@ -281,4 +293,52 @@ export class MessageModel extends DocumentDbModel<
       query: "SELECT * FROM messages m WHERE (m.fiscalCode = @fiscalCode)"
     });
   }
+
+  public async attachStoredContent(
+    blobService: BlobService,
+    messageId: string,
+    partitionKey: string,
+    message: IMessageWithContent
+  ): Promise<Either<Error, Option<DocumentDb.AttachmentMeta>>> {
+    // this is the attachment id __and__ the filename
+    const blobId = this.getMessageAttachmentName(messageId);
+
+    // store media (attachment) with message content in blob storage
+    const errorOrMessageContent = await upsertBlobFromText(
+      blobService,
+      MESSAGE_BLOB_STORAGE_CONTAINER_NAME,
+      blobId,
+      JSON.stringify({ id: messageId, ...message })
```

Comment:
is the following getting stored in the blob? (full message + its ID) ? we should just store the `content`, that's it. 

```json
{
  "id": "...",
  "message": {
    ...,
    "content": {
      ...
    }
  }
}
```

Reply:
ok